sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -109,6 +110,32 @@ def __init__(self, endpoint, credential, **kwargs):
         self._string_code_unit = None if kwargs.get("api_version") == "v3.0" else "UnicodeCodePoint"
         self._deserialize = _get_deserialize()
 
+
+    def invoke(self, request, **kwargs):
+
+        # This part is because I don't want to force people to give the full URL. This is open to debate
```

Comment:
Supporting a relative URL seem to be reasonable. It would not be a stretch for me to assume that this would be interpreted as relative to the base specified when constructing the client.